webapp/src/ts/services/feedback.service.ts

```diff
@@ -30,9 +31,26 @@ export class FeedbackService {
   };
 
   private logIdx = 0;
+  private lastErrorMessage;
   private readonly LEVELS = ['error', 'warn', 'log', 'info'];
   private readonly LOG_LENGTH = 20;
   private readonly logCircularBuffer = new Array(this.LOG_LENGTH);
+  // List of Error messages to not automatically log to feedback
+  // Can be a lower-cased partial string or a regular expression
+  private readonly NO_FEEDBACK_MESSAGES = [
+    'failed to fetch',
```

Comment:
Change this to the equivalent regex with `/i` then we don't need to lowercase the error messages and the code gets cleaner because they're all regexes.